lerobot/common/logger.py

```diff
@@ -47,9 +47,14 @@ def log_output_dir(out_dir):
 
 def cfg_to_group(cfg: TrainPipelineConfig, return_list: bool = False) -> list[str] | str:
     """Return a group name for logging. Optionally returns group name as list."""
+    # TODO: these were used to support multirepodataset in the past - think how they could be supported in new way?
+    dataset_tag = cfg.dataset.repo_id
+    if dataset_tag.startswith('['):
+        tags = dataset_tag.strip('[]').split(',')
+        dataset_tag = f"{tags[0].strip()}_and_more"
```

Comment:
some of these names were too long so I just use the first name and add some stuff

feel free to change this or remove if you don't think it's a big enough problem